lib/api/WP_Auth0_Api_Abstract.php

```diff
@@ -180,47 +182,37 @@ protected function set_path( $path ) {
 	 */
 	protected function set_bearer( $scope ) {
 
-		$this->api_token = wp_cache_get( self::CACHE_KEY, WPA0_CACHE_GROUP );
-		if ( ! $this->api_token ) {
-			$this->api_token = $this->options->get( 'auth0_app_token' ); // REMOVE
-		}
+		$api_token = WP_Auth0_Api_Client_Credentials::get_stored_token();
```

Comment:
What's the reason to have this method static and the `call()` one part of the instance instead?

Reply:
`call()` is part of an abstract so it needs to be defined. This can accept parameters required for the API call. In this case, no parameters are needed but other child classes use that. 

The static methods don't require any parameters and act on global state (database).